src/main/java/edu/unc/lib/boxc/migration/cdm/services/ArchivalDestinationsService.java

```diff
@@ -144,7 +148,7 @@ public void addArchivalCollectionMappings(DestinationMappingOptions options) thr
                 }
             }
         } else {
-            throw new Exception("Field option is empty");
+            throw new IllegalArgumentException("Field option is empty");
```

Comment:
can `public void addArchivalCollectionMappings(DestinationMappingOptions options) throws Exception {` be changed to something else now? Probably `public void addArchivalCollectionMappings(DestinationMappingOptions options) throws IOException {` at least